package.json

```diff
@@ -37,7 +37,8 @@
   "scripts": {
     "fmt": "semistandard --fix",
     "lint": "semistandard --verbose | snazzy",
-    "test": "npm run lint && mocha test"
+    "test": "npm run lint && mocha test",
+    "cov": "istanbul cover ./node_modules/.bin/_mocha test"
```

Comment:
Taking coverage is very good practice, but not related to this PR. Would you remove this line from this PR?

FYI: I'm using [Coveralls](https://coveralls.io/)